source/text/label.ts

```diff
@@ -52,35 +228,50 @@ export class Label {
         return Text.DEFAULT_LINE_FEED;
     }
 
+    /**
+     * If enabled, breaks lines automatically at line width (while typesetting). Note that elide mode takes precedence.
+     */
     set wrap(flag: boolean) {
         this._wrap = flag;
     }
     get wrap(): boolean {
         return this._wrap;
     }
 
+    /**
+     * If enabled, shrinks the label to line width. Depending on the elide mode, the ellipses is put left, middle, or
+     * right. The ellipsis string can be adjusted (@see {@link ellipsis}). If the labels text does not exceed the line
+     * width no elide will be applied.
+     */
     set elide(elide: Label.Elide) {
         this._elide = elide;
     }
     get elide(): Label.Elide {
         return this._elide;
     }
 
+    /**
+     * Allows to override/customize the ellipsis string used for text elide (@see {@link elide}).
+     */
     set ellipsis(ellipsis: string) {
         this._ellipsis = ellipsis;
     }
     get ellipsis(): string {
         return this._ellipsis;
     }
 
-
+    /**
+     * Line width used to either maximum length for elide or maximum length for line breaks due to word wrap. The line
+     * width is expected in typesetting space (the unit used while Typesetting, i.e., the unit as the font face's glyph
+     * texture atlas).
```

Comment:
@cgcostume 
This is wrong documented. The lineWidth setter does *not* expect the line width in typesetting space, but in the space unit used for font size and positioning. (Have a look at the lineWidth getter implementation and documentation).